homeassistant/components/apple_tv/__init__.py

```diff
@@ -353,6 +353,7 @@ async def _async_setup_device_registry(self):
 
             attrs["model"] = "Apple TV " + dev_info.model.name.replace("Gen", "")
             attrs["sw_version"] = dev_info.version
+            attrs["suggested_area"] = self.atv.name
```

Comment:
There isn't a `name` property here as its actually a different object than the one in the config flow.  I need to figure out how to get the name with that we have.

Reply:
I suggest that you fetch it from the config entry directly (see `name` in `attrs` higher up in the method).